pipeline/tests/powdr_std.rs

```diff
@@ -250,3 +250,8 @@ fn sort() {
         assert_eq!(input_sorted, result);
     }
 }
+#[test]
+fn btree() {
+    let f = "std/btree_test.asm";
+    verify_test_file(f, Default::default(), vec![]).unwrap();
```

Comment:
I guess this is how I should do it in #1310 as well!

Reply:
I mean not sure. It is nice to have tests in the same file. How does rust do it?

Reply:
Just checked for `BTreeMap`:
- In `src/collections/btree/map.rs`, they have `#[cfg(tests] mod tests;`
- The actual tests are in `src/collections/btree/map/tests.rs`

So like any rust lib. I guess the difference with us that we don't have an annotation like `#[cfg(tests]` (yet).